terraform/deployments/search-api-v2/discovery_engine.tf

```diff
@@ -0,0 +1,96 @@
+# Creates and configures an unstructured datastore for Google Discovery Engine ("Vertex AI Search")
+# see https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/
+
+module "govuk_content_discovery_engine" {
+  source = "../modules/google_discovery_engine_restapi"
```

Comment:
This depends on the [`discovery_engine_restapi` module](https://github.com/alphagov/search-v2-infrastructure/tree/main/terraform/modules/google_discovery_engine_restapi) that will also need migrating at the same time 💡